packages/flutter/test/cupertino/route_test.dart

```diff
@@ -259,17 +258,16 @@ void main() {
     final GlobalKey scaffoldKey = GlobalKey();
 
     await tester.pumpWidget(
```

Comment:
Please add this comment to the top of the test: 
`// Regression test for https://github.com/flutter/flutter/issues/28728`